src/DynamoCore/Search/SearchElements/NodeSearchElement.cs

```diff
@@ -13,14 +13,13 @@ namespace Dynamo.Search.SearchElements
     /// <summary>
     ///     Base class for all Dynamo Node search elements.
     /// </summary>
-    public abstract class NodeSearchElement : INotifyPropertyChanged, ISearchEntry, ISource<NodeModel>
+    public abstract class NodeSearchElement : ISearchEntry, ISource<NodeModel>
```

Comment:
Just to make sure this removal is safe, you need to confirm that these properties are only ever set once (during construction) and they do not change subsequently. If they do change (e.g. renaming the `name` or `description` of a custom node), the view needs to be updated.